src/mbgl/renderer/tile_pyramid.cpp

```diff
@@ -58,7 +58,8 @@ void TilePyramid::update(const std::vector<Immutable<style::LayerProperties>>& l
                          const Range<uint8_t> zoomRange,
                          optional<LatLngBounds> bounds,
                          std::function<std::unique_ptr<Tile>(const OverscaledTileID&)> createTile,
-                         optional<uint8_t> sourcePrefetchZoomDelta) {
+                         const optional<uint8_t>& sourcePrefetchZoomDelta,
+                         const optional<uint8_t>& maxOverscaleFactor) {
```

Comment:
Thinking about naming consistency, since we have `sourcePrefetchZoomDelta` and `maxOverscaleFactor` is configurable by source, would a better name here be `sourceMaxOverscaleFactor`?